cumulus/parachains/runtimes/assets/asset-hub-rococo/src/genesis_config_presets.rs

```diff
@@ -0,0 +1,203 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! # Asset Hub Rococo Runtime genesis config presets
+
+#[cfg(not(feature = "std"))]
+use crate::vec;
+use crate::Vec;
+use cumulus_primitives_core::ParaId;
+use hex_literal::hex;
+use parachains_common::{AccountId, AuraId, Balance as AssetHubBalance};
+use sp_core::{crypto::UncheckedInto, sr25519};
+#[cfg(not(feature = "std"))]
+use sp_std::alloc::format;
+
+const ASSET_HUB_ROCOCO_ED: AssetHubBalance = crate::ExistentialDeposit::get();
+
+//---------------------------------------------------------------
+// copied from: cumulus/polkadot-parachain/src/chain_spec/mod.rs
```

Comment:
Probably we would need some shared crate here, as this will be duplicated in every runtime.

Reply:
for cumulus testnet parachains, we already have shared crate with constants: https://github.com/paritytech/polkadot-sdk/tree/master/cumulus/parachains/runtimes/constants

Reply:
I moved shared code to:
https://github.com/paritytech/polkadot-sdk/blob/f3e7e4d7cc90ed5bf7d4062cacf670ab40da2122/cumulus/parachains/common/src/genesis_config_helpers.rs

Currently node has its own copy, but it is gonna be removed as we migrate all chainspecs to presets.